OasisDEX/oasis-borrow
# Stop loss open multiply flow

# [Stop loss open multiply flow](https://app.shortcut.com/oazo-apps/story/4151/stop-loss-form-integration-in-multiply-vault) 
(and all related stories except for unit tests)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- added stop loss to multiply open flow
  
## How to test 🧪
  <Please explain how to test your changes>
- use NewComponents and StopLossOpenFlow feature toggle and try to open multiply vault with and without stop loss (when added during vault open flow, SL should be enabled in manage vault protection tab)
